test/linter/test-versions.js

```diff
@@ -72,6 +72,48 @@ function removedAfterAdded(statement) {
   );
 }
 
+/**
+ * @param {VersionValue} xver
+ * @param {VersionValue} yver
+ */
+function compareVersionNumber(xver, yver) {
```

Comment:
This will need to handle "preview" as well, which didn't exist when you started this lint. That also makes "number" in the name inaccurate.